src/Components/v2/Carousel.tsx

```diff
@@ -45,16 +72,17 @@ export const LargeCarousel = (props: Props) => {
     infinite: false,
     speed: 500,
     variableWidth: true,
-    slidesToScroll: 5,
+    slidesToScroll: SLIDES_TO_SCROLL,
     ...props.settings,
+    afterChange: currentIndex => props.afterChange(currentIndex),
   }
 
   let slickRef = null
 
   const LeftArrow = () => {
     return (
       <ArrowButton
-        left={-8}
+        left={-35}
```

Comment:
Why change this? This will have cascading effects on other uses. If positioning needs to be adjusted on a one-off basis can be handled in the `renderArrow` callbacks.

Reply:
The arrows need to be positioned `absolute` otherwise when they are rendered and then not rendered they shift the entire carousel. I need to updated the `left` & `right` after changing to `absolute`

Reply:
Ah yeah that makes sense, though all of this can take place in the callbacks; see https://github.com/artsy/reaction/blob/master/src/Components/v2/__stories__/Carousel.story.tsx#L91-L161 for an example.